addons/web/static/src/views/fields/properties/properties_field.js

```diff
@@ -143,11 +143,6 @@ export class PropertiesField extends Component {
      * @returns {boolean}
      */
     checkPopoverClose(target) {
-        if (document.activeElement.closest(".o_field_property_definition")) {
```

Comment:
So this was useless?

Reply:
Maybe elaborate a bit more on this on the commit message?

Right now it explains what it solves but not really why removing that code bit is ok.